microsoft-edge/webview2/concepts/iframes.md

```diff
@@ -0,0 +1,323 @@
+
+---
+
+
+<!-- ------------------------------ -->
+#### Example of running JavaScript code in iFrames by using ExecuteScript
+
+<!-- link to API Ref that contains an example code listing? -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+```javascript
+
+```
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+```javascript
+
+```
+
+##### [Win32/C++](#tab/win32cpp)
+
+```javascript
+
+```
+
+---
+
+
+<!-- ====================================================================== -->
+## Challenges when working with iFrames
+
+iFrames use the same navigation states as other WebView2 pages.  All navigation events except `SourceChanged` events, currently, are supported for iFrames the same way as in WebView2.
+
+The article [Navigation events for WebView2 apps](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/navigation-events) is valid for frames as well as webpages.
+
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2.NavigationStarting Event](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.navigationstarting?view=webview2-dotnet-1.0.1370.28)
+* [CoreWebView2Frame.NavigationStarting Event](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2frame.navigationstarting?view=webview2-dotnet-1.0.1370.28)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2::FrameNavigationStarting event (add](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2?view=webview2-1.0.1293.44#add_framenavigationstarting), [remove)](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2?view=webview2-1.0.1293.44#remove_framenavigationstarting)
+* [ICoreWebView2Frame2::NavigationStarting event (add](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2frame2?view=webview2-1.0.1293.44#add_navigationstarting), [remove)](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2frame2?view=webview2-1.0.1293.44#remove_navigationstarting)
+
+* [ICoreWebView2::FrameNavigationCompleted event (add](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2?view=webview2-1.0.1293.44#add_framenavigationcompleted), [remove)](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2?view=webview2-1.0.1293.44#remove_framenavigationcompleted)
+* [ICoreWebView2Frame2::NavigationCompleted event (add](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2frame2?view=webview2-1.0.1293.44#add_navigationcompleted), [remove)](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2frame2?view=webview2-1.0.1293.44#remove_navigationcompleted)
+
+`ConvertWebView2ToFrame`
+<!-- TODO: is there an API Ref link? -->
+
+---
+
+
+<!-- ====================================================================== -->
+## API Reference overview
+
```

Comment:
ok to delete section, it's dup & doesn't scale.   it was easy to copy that section from https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?tabs=dotnetcsharp#iframes but it's more important to construct tab-sets within topical h2 sections (that's a must-have) than to have & maintain this section here at end of article.